### PR TITLE
Improve highlights ux, add star to query

### PR DESCRIPTION
**Pull request by thostetler**
Adds `*` to `hl.fl` field for highlights
Adds a `no highlights` message when none are returned

image

### CODE REVIEW

File: src/js/widgets/list_of_things/widget.js [hunk 1]
```diff
@@ -606,7 +606,7 @@ define([
           var q = this.model.get('currentQuery').clone();
           q.set({
             hl: 'true',
-            'hl.fl': 'title,abstract,body,ack',
+            'hl.fl': 'title,abstract,body,ack,*',
```
Comment (romanchyla) [inline src/js/widgets/list_of_things/widget.js:609]: i think it is actually better to just say: `'hl.fl': '*'`

**Review by romanchyla — APPROVED** [review]

File: src/js/widgets/list_of_things/widget.js (hunk 1 above)
Comment (thostetler) [reply src/js/widgets/list_of_things/widget.js:609]: I'll check it out, but doing '*' returned an empty param and causes an error in the response parser.  I didn't look into it at all though, just figured they were equivalent

Comment (thostetler) [reply src/js/widgets/list_of_things/widget.js:609]: doing `hl.fl: '*'` 
image

gets an empty response: 
image

That is causing an error.  Anything I can do about that? is that correct behavior

**Comment by thostetler** [comment]
I'm going to merge this, we can circle back around and make the change to `*` later.  Assuming the changes are equivalent.

**Pull request merged**